packages/newspringchurchapp/src/ui/MyPrayer/MyPrayerCard.tests.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Providers from 'newspringchurchapp/src/Providers';
+
+import MyPrayerCard from './MyPrayerCard';
+
+const prayer1 = {
```

Comment:
If all of these were in one `const` set to an array or prayers, you would only need one test.